rdrf/rdrf/registration_rdrf.py

```diff
@@ -24,4 +24,29 @@ def post(self, request, *args, **kwargs):
     def get_context_data(self, **kwargs):
         context = super(RdrfRegistrationView, self).get_context_data(**kwargs)
         context['registry_code'] = self.registry_code
+        context["preferred_languages"] = self._get_preferred_languages()
         return context
+
+    def _get_preferred_languages(self):
+        # Registration allows choice of preferred language
+        # But we allow different sites to expose different values
+        # over time without code change via env --> settings
+        from django.conf import settings
+
+        # The default list is english only which we don't bother to show
+        if len(settings.LANGUAGES) == 1:
+            if settings.LANGUAGES[0][0].lower() == "en":
+                return []
+            
+        l = [] 
+
+        class LanguageWrapper:
```

Comment:
using `collections.namedtuple` would be simpler. 
I would also move this to a module (like  `models.py`) so you can re-use it in the validation code below, that currently uses slicing instead of field names.